css/components/button.css

```diff
@@ -0,0 +1,89 @@
+/* === css for button component === */
+
+button{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: #fff;
+    padding: .7rem 1.3rem;
+    font-family: inherit;
+    font-size: var(--normal-font-size);
+    font-weight: var(--font-medium);
+    border: none;
+    border-radius: 5px;
+    width: 9rem;
+    margin-bottom: var(--mb-0-5);
+    cursor: pointer;
+}
```

Comment:
Don't style on button tag directly. Use some class as `.btn`

Reply:
ok will add that feature.